src/config/supported-keys.json

```diff
@@ -0,0 +1,990 @@
+{
+    "version": "1.1.0",
+    "keys": {
+        "crypto.BTC.address": {
+            "deprecatedKeyName": "BTC",
+            "deprecated": false,
+            "validationRegex": "^(bc1|[13])[a-zA-HJ-NP-Z0-9]{25,39}$"
+        },
+        "crypto.ETH.address": {
+            "deprecatedKeyName": "ETH",
```

Comment:
Why do we have them equal, even if they haven't been deprecated?

Reply:
I'd imagine `deprecatedKeyName` being set to `null` if it's equal to the current key name.